src/components/PurchaseOrder/PODetails/PODetailsView.js

```diff
@@ -159,6 +159,14 @@ class PODetailsView extends Component {
               <FolioFormattedTime dateString={get(order, 'metadata.createdDate')} />
             </KeyValue>
           </Col>
+          <Col
+            xs={6}
+            lg={3}
+          >
+            <KeyValue label={<FormattedMessage id="ui-orders.orderDetails.dateOrdered" />}>
+              <FolioFormattedTime dateString={get(order, 'dateOrdered')} />
```

Comment:
`order?.dateOrdered`. You don't need `get`

Reply:
Should it be displayed only for Open orders?

Reply:
Looks like yes, updated